---
title: Der Throw-Operand muss von System.Exception abgeleitet werden.
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: abaef1ac3572da1c8270bdd4717876d9b97c91e7
ms.sourcegitcommit: bef803e2025642df39f2f1e046767d89031e0304
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 02/15/2019
ms.locfileid: "56303230"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>Der Throw-Operand muss von System.Exception abgeleitet werden.
Das für `Throw` angegebene Argument muss entweder eine Instanz von `System.Exception` oder eine Instanz einer von `System.Exception`abgeleiteten Klasse sein.  
  
 **Fehler-ID:** BC30665  
  
## <a name="to-correct-this-error"></a>So beheben Sie diesen Fehler  
  
-   Verwenden Sie ein Argument, das von `System.Exception`abgeleitet wird, wie im folgenden Beispiel gezeigt.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Siehe auch
- [Throw-Anweisung](../../visual-basic/language-reference/statements/throw-statement.md)
- [Try...Catch...Finally-Anweisung](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [Behandeln und Auslösen von Ausnahmen in .NET](../../standard/exceptions/index.md)
